CONTRIBUTING.md

```diff
@@ -122,7 +122,13 @@ The testing process for Hubs is mostly a manual one. You need to test your chang
 
 The Hubs team has a more in-depth testing and release process internally, but we don't have any additional testing process for external contributors at this time.
 
-### 5. High Level Project Organization
+
+### 5. Modifying The 2-D Sprite Graphics
+
+To update the 2-D sprite graphics, you must (manually) install [ImageMagick](https://imagemagick.org/script/download.php) to your command line.  See `doc/spritesheet-generation.md` for further details
```

Comment:
We may want to remove the last paragraph of the spritesheet-generation.md file referring to Figma because it's not really relevant to external contributors.

Reply:
I'll leave that to people more familiar with Figma and the details of spritesheet generation.